package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -331,9 +339,10 @@ def produce_coordinate_arrays_single_process(topology_file_path, trajectory_file
     return (start_frame_relevant_particle_coordinate_array_xyz, end_frame_relevant_particle_coordinate_array_xyz)
 
 
-def generate_streamlines_3d(topology_file_path, trajectory_file_path, grid_spacing, MDA_selection, start_frame,
-                            end_frame, xmin, xmax, ymin, ymax, zmin, zmax, maximum_delta_magnitude=2.0,
-                            num_cores='maximum'):
+def generate_streamlines_3d(topology_file_path: str, trajectory_file_path: str, grid_spacing: float, MDA_selection: str,
+                            start_frame: int, end_frame: int, xmin: float, xmax: float, ymin: float, ymax: float,
+                            zmin: float, zmax: float, maximum_delta_magnitude: float=2.0,
+                            num_cores: int ='maximum') -> Tuple[Sequence[float], Sequence[float], Sequence[float]]:
```

Comment:
```suggestion
def generate_streamlines_3d(
    topology_file_path: str, trajectory_file_path: str,
    grid_spacing: float, MDA_selection: str,
    start_frame: int, end_frame: int,
    xmin: float, xmax: float,
    ymin: float, ymax: float,
    zmin: float, zmax: float,
    maximum_delta_magnitude: float=2.0,
    num_cores: int ='maximum'
) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
```

I fixed the return annotation.